config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  #root 'users#index'
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far! Should  have `users` and `posts` routes added to `config/routes.rb` and maintained a `RESTful URL` as `:Index | :show` 🙏🏼 

```ruby
# config/routes.rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Fixed.